### PR TITLE
Tpu support

### PR DESCRIPTION
**Pull request by muhammetfaik**
I think create update for tpu and with testing vgg2dataset.

### CODE REVIEW

**Comment by muhammetfaik** [comment]
Ebenin amı

**Comment by muhammetfaik** [comment]
Hahahahaha nice joke motherfucker

**Comment by muhammetfaik** [comment]
ebenin amı (you can translate turkish)

**Comment by muhammetfaik** [comment]
ebenin amı(You can translate Turkish)

File: vgg2dataset.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    r = session.get(FILE_URL, data=payload, stream=True)
+    bytes_written = 0
+    for data in r.iter_content(chunk_size=4096):
+        f.write(data)
+        bytes_written += len(data)
+        MiB = bytes_written / (1024 * 1024)
+        sys.stdout.write("\r{MiB:0.2f} MiB downloaded...")
+        sys.stdout.flush()
+
+print("\nDone.")
```
Comment (tdh8316) [inline vgg2dataset.py]: Need a new line to resolve the pep8 Insp.

**Comment by bryanlyon** [comment]
While we would welcome a PR with TPU support, this PR does not provide that.  What it does provide -- A downloader for the VGGFace2 dataset is encumbered with limitations and license restrictions while providing basically no benefit to the majority of our users.

For these reasons I'm going to close this PR.

If you want to try again and submit a PR to add TPU support, we will welcome it, but please do not mix in unnecessary features or side packages like VGGFace2.

**Pull request closed without merging**